<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calibration Compare</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;Image Info&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Width</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Raw Size</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_mouse">
             <property name="text">
              <string>Pos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Manual Point</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Point 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Point 2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Point 3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Point 4</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_add">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_clear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="6">
              <widget class="QLineEdit" name="lineEdit_y3"/>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="lineEdit_x4"/>
             </item>
             <item row="1" column="5">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Y2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="5">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Y3</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Y4</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Y1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_x2"/>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>X2</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>X1</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>X4</string>
               </property>
              </widget>
             </item>
             <item row="3" column="6">
              <widget class="QLineEdit" name="lineEdit_y4"/>
             </item>
             <item row="1" column="6">
              <widget class="QLineEdit" name="lineEdit_y2"/>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_x3"/>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>X3</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_x1"/>
             </item>
             <item row="0" column="6">
              <widget class="QLineEdit" name="lineEdit_y1"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Auto Point</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>-</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Point 1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_ax3"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>X3</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_ax2"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>X1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>X2</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_ay2"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>X4</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_ay3"/>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_ay1"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_ax1"/>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="lineEdit_ay4"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_ax4"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Y1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Y2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Y3</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_32">
               <property name="text">
                <string>Y4</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Error</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableWidget"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_N"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionExit_X"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_N">
   <property name="text">
    <string>New(N)</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>Save(S)</string>
   </property>
  </action>
  <action name="actionExit_X">
   <property name="text">
    <string>Exit(X)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
